Extensions/old_blue.css

```diff
@@ -12,1047 +12,225 @@
-}
-
-.is-followed.with-commentary .post-activity-note-content {
-	border-color: #e5e7ea;
-}
-
-.is-followed .post-activity-note-content .note-text {
-	color: #86909e;
-}
-
-.is-followed .post-activity-note-content .note-text .note-text-link {
-	color: var(--old-blue);
-}
-
-.is-followed .post-activity-note-content .note-added-tags,
-.is-followed .post-activity-note-content .note-added-text {
-	color: var(--old-blue);
-}
-
-.post-activity-note-content .note-added-tags .tag,
-.post-activity-note-content .note-added-text .tag {
+.chrome.white[disabled],
+.chrome.white[disabled]:active {
 	color: var(--old-grey);
 }
 
-.post-activity-reply .submit {
-	color: var(--old-accent);
-}
-
-.post-activity-header .subscription-status .outer {
-	fill: var(--old-blue);
+.chrome.green {
+	border-color: var(--old-green);
+	background: var(--old-green);
 }
 
-.post-activity-header .subscription-status .bolt {
-	fill: var(--old-blue);
+.chrome.green.disabled,
+.chrome.green.disabled:active,
+.chrome.green[disabled],
+.chrome.green[disabled]:active {
+	background: var(--old-green);
 }
 
-.post-activity-header .subscription-status.is-subscribed .inner {
-	fill: var(--old-blue);
+.chrome.red,
+.chrome.red.ui_button {
+	border-color: var(--old-red);
+	background: var(--old-red);
 }
 
-.post-forms-glass {
-	background-color: rgba(54,70,93,.95);
+.chrome.red.disabled,
+.chrome.red.disabled:active,
+.chrome.red[disabled],
+.chrome.red[disabled]:active {
+	background: var(--old-red);
 }
 
-.post-form--form.blocked {
-	background-color: var(--old-accent);
+.chrome.blue {
+	border-color: var(--old-accent);
+	background: var(--old-accent);
 }
 
-.post-container {
-	color: var(--old-text);
+.chrome.blue.disabled,
+.chrome.blue.disabled:active,
+.chrome.blue[disabled],
+.chrome.blue[disabled]:active {
+	background: var(--old-accent);
 }
 
-.post-container a {
-	color: var(--old-text);
+.chrome.other_blue {
+	border-color: var(--old-accent);
+	background: var(--old-accent);
 }
 
-.post-form--tag-editor .tag-label {
+.sponsored_label {
 	color: var(--old-grey);
 }
 
-.post-form--tag-editor .tag-label.hover-style,
-.post-form--tag-editor .tag-label.selected {
-	color: var(--old-accent);
-}
-
-.pop-menu input[type=text].field,
-.post-form input[type=text].field {
-	color: var(--old-text);
-}
-
-.media-container .video-error {
-	background-color: var(--old-vidgrey);
+#tab_switching .tab_blog .tab_notice {
+	background-color: var(--old-accent);
 }
 
-.media-container.audio-media-container .album-art-uploader .media-dropzone {
-	color: var(--old-purple);
+.tab_post_type .post_type_icon {
+	background-image: url("https://new-xkit.github.io/XKit/Extensions/dist/quick_compose_sprite.png");
 }
 
-.media-dropzone .media-url {
-	color: var(--old-text);
+@media only screen and (-webkit-min-device-pixel-ratio: 2), only screen and (-moz-min-device-pixel-ratio: 2), only screen and (min-device-pixel-ratio: 2), only screen and (min-resolution: 192dpi), only screen and (min-resolution: 2dppx) {
```

Comment:
don't we already got one of these? or is this a different media query or smth?

Reply:
different media query